packages/react/tsconfig.json

```diff
@@ -2,10 +2,13 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "isolatedModules": false,
-    "outDir": "./dist"
+    "outDir": "./dist",
+    "types": ["jest"]
+
   },
   "include": [
-    "./src/**/*.ts"
+    "./src/**/*.ts",
+    "./src/**/*.tsx"
```

Comment:
Are u sure about this change? it will affect tsconfig not only for jest. IMO looks like it is necessary